helm/prometheus-rules/templates/alerting-rules/certificate.workload-cluster.rules.yml

```diff
@@ -16,12 +16,13 @@ spec:
       annotations:
         description: '{{`Certificate {{ $labels.path }} on {{ $labels.node }} will expire in less than a month.`}}'
         opsrecipe: renew-certificates/
+        dashboard: https://giantswarm.grafana.net/d/a2f4976Zk/certificates?orgId=1
```

Comment:
Prefer to link the dashboard UID/title like explained here:
https://github.com/giantswarm/prometheus-rules/blob/master/README.md?plain=1#L49

Example:
https://github.com/giantswarm/prometheus-rules/blob/master/helm/prometheus-rules/templates/alerting-rules/prometheus-agent.rules.yml#L20

Reply:
Does that hold for dashboards in Grafana Cloud as well? I thought that section only applied for dashboards in the MC-specific Grafana

Reply:
Well, you're right :-)